sormas-backend/src/main/java/de/symeda/sormas/backend/importexport/ImportFacadeEjb.java

```diff
@@ -164,6 +194,26 @@ public void generateCaseImportTemplateFile() throws IOException {
 		writeTemplate(Paths.get(getCaseImportTemplateFilePath()), importColumns, true);
 	}
 
+	@Override
+	public void generateEventImportTemplateFile() throws IOException {
+
+		createExportDirectoryIfNecessary();
+
+		char separator = configFacade.getCsvSeparator();
+
+		List<ImportColumn> importColumns = new ArrayList<>();
+		appendListOfFields(importColumns, EventDto.class, "", separator);
+		importColumns.add(ImportColumn.from(EventParticipantDto.class, EventParticipantDto.INVOLVEMENT_DESCRIPTION, String.class, separator));
```

Comment:
I think at least the region and district fields of the event participant also need to be part of the import file

Reply:
Done.
A remark by the way, the Event Participant import (the one available from the Event Participant grid) didn't provide a mean to do that. If you want I can update the Event Participant import too to make these 2 properties available.

Reply:
That would be really helpful, thank you!

Reply:
Done